casepro/settings_production.py

```diff
@@ -151,9 +151,17 @@ def parse_channel_info(data):
 PODS = [{
     'label': "family_connect_registration_pod",
     'title': "Registration Information",
-    'url': os.environ.get('REGISTRATION_URL', ''),
-    'token': os.environ.get('REGISTRATION_AUTH_TOKEN',
-                            'replace-with-auth-token'),
+    'hub_api_url': os.environ.get('REGISTRATION_URL', ''),
+    'hub_token': os.environ.get('REGISTRATION_AUTH_TOKEN',
+                                'replace-with-auth-token'),
+    'identity_store_api_url': os.environ.get(
+        'IDENTITY_STORE_API_URL',
+        'https://identity-store/api/v1/',
+    ),
+    'identity_store_token': os.environ.get(
+        'IDENTITY_STORE_TOKEN',
```

Comment:
Maybe this should be `IDENTITY_STORE_AUTH_TOKEN`, to make it match with the existing env var for the registration service?

Reply:
Updated, thanks!

Reply:
Aren't the settings currently `IDENTITY_AUTH_TOKEN` and `IDENTITY_API_ROOT` ?

Reply:
Oh cool we have these settings already in the CasePro app. That makes this easier.